Dockerfile

```diff
@@ -36,7 +36,9 @@ RUN apt-get update \
     make \
     tar \
     xxd \
+    cmake \
     && git config --global --add safe.directory /src \

```

Comment:
We should remove this line.

Reply:
Sorry, line 40. Not line 39. GitHub messed this up.

Reply:
To be clear, you're referring to removing "/src" as a git safe directory, correct?
/usr/src is the mount point of the repo, and needs to be marked safe for git to function in the devcontainer

Reply:
Yeah, we shouldn't need `/src` to be added to `safe.directory`. Only `/usr/src`.